extensions/resource-deployment/package.nls.json

```diff
@@ -46,5 +46,28 @@
 	"bdc-agreement-azdata-eula": "azdata License Terms",
 	"bdc-agreement-bdc-eula": "SQL Server License Terms",
 	"deployment.configuration.title": "Deployment configuration",
-	"azdata-install-location-description": "Location of the azdata package used for the install command"
+	"azdata-install-location-description": "Location of the azdata package used for the install command",
+	"azure-sqlvm-display-name": "SQL Server on Azure Virtual Machine",
+	"azure-sqlvm-description": "Create SQL virtual machines on Azure. Best for migrations and applications requiring OS-level access. Lift-and-shift ready.",
+	"azure-sqlvm-deploy-dialog-title": "Deploy Azure SQL virtual machine",
+	"azure-sqlvm-deploy-dialog-action-text": "Script to notebook",
+	"azure-sqlvm-agreement": "I accept {0}, {1} and {2}.",
+	"azure-sqlvm-agreement-sqlvm-eula": "Azure SQL VM License Terms",
+	"azure-sqlvm-agreement-azdata-eula": "azdata License Terms",
```

Comment:
Sync up with @ranasaria - we're removing the azdata EULA acceptance in favor of using the azdata extension (which will handle EULA acceptance)

Reply:
This is going in the current release. I can probably remove this for the next release.